source/openapi3/split.yaml

```
@@ -3290,6 +3290,154 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GetUserDetailsResponse'
+  /webhooks:
+    get:
+      tags:
+        - Webhooks
+      summary: List all webhooks
+      description: ''
+      operationId: GetWebhooks
+      parameters: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ListAllWebhooksResponse'
+        '404':
+          description: Not Found
+
+  '/webhooks/{webhook_id}/deliveries':
+    get:
+      tags:
+        - Webhooks
+      summary: List deliveries for a webhook
+      description: 'NOTE: There is a historical limit of 30 days for webhook deliveries.'
```

"historical limit" kind of implies that there used to be a limit, but there might not be one any more.

```suggestion
      description: 'NOTE: This is limited to webhook deliveries created in the last 30 days.'
```